Dockerfile

```diff
@@ -15,7 +15,7 @@ RUN go build -a -trimpath -ldflags "-s -w -X go.k6.io/k6/lib/consts.VersionDetai
 
 # Runtime stage
 FROM alpine:3.17
-RUN apk add --no-cache ca-certificates=~20220614 && \
+RUN apk add --no-cache ca-certificates && \
```

Comment:
Maybe let's add an [ignore rule for Hadolint](https://github.com/hadolint/hadolint/blob/dc66e0996474b324f10649618f424fc76ca60e28/README.md#inline-ignores)?

Reply:
I researched for it and I missed it, thanks for sharing